### ISSUE TITLE
Puavo menu Dark mode support

### ISSUE THREAD

**Issue by asokero** [puavo-org/puavo-os#670]
Puavo Menu does not recognize the light/dark theme changes in current Bookworm Gnome and therefore stays unaffected if this option is used. Puavo menu should support the Gnome UI-style options.

**Comment by sarrchri** [comment]
Here's a sum up of my findings from what I've looked at this a little bit.

Puavomenu theme *seems* to be affected by key `gtk-theme` at `org.gnome.desktop.interface`. I found no direct mention of this anywhere in puavomenu code, but manually changing the value of `gtk-theme`, changes puavomenu theme accordingly. (!! The key values are case sensitive.)
`gsettings set org.gnome.desktop.interface gtk-theme 'Arc-Dark'`
`gsettings set org.gnome.desktop.interface gtk-theme 'Arc'`

Gnome Settings Light/Dark theme selection controls key `color-scheme` at `org.gnome.desktop.interface`, it's possible values are as follows.
```
"default" // (/usr/share/themes/Default ?)
"prefer-light"
"prefer-dark"
```
Also, the value of `color-scheme` seems to affect the value of `gtk-theme` like so:
If Dark theme is selected from Gnome Settings (thus setting `color-scheme` to `prefer-dark`), then whatever theme is selected for `gtk-theme`, it is changed to include `-Dark`.
This gets reverted if Default or Light theme is selected.

For example, theme `Arc`.
```
Light:
color-scheme = "default" OR "prefer-light"
gtk-theme = "Arc" // no changes happen

Dark:
color-scheme = "prefer-dark"
gtk-theme = "Arc" --> "Arc-Dark"
```
